packages/smooth_app/lib/pages/preferences/user_preferences_connect.dart

```diff
@@ -102,7 +102,7 @@ class UserPreferencesConnect extends AbstractUserPreferences {
 
             final Email email = Email(
               body: await _emailBody,
-              subject: 'Smoothie help',
+              subject: appLocalizations.help_with_openfoodfacts,
```

Comment:
Perhaps that would make sense to add here the hard-coded string ` (Help with Open Food Facts)` for email rule.

Reply:
There was this suggestion left by pierre, so did the changes accordingly

Reply:
yes, my idea was that we should keep the original hardcoded to avoid having to set 150 email rules

Reply:
> yes, my idea was that we should keep the original hardcoded to avoid having to set 150 email rules

So is it to leave the string untranslated here?

Reply:
Translated string (Untranslated source string)

Reply:
I'm getting confused what exactly to do, 
Should i leave the pr as it is now ? 
- [ ]